.github/workflows/china-deploy-static-websites-dev.yaml

```
@@ -1,131 +1,131 @@
-name: China Cloud Deploy Static Websites (Dev)
+# name: China Cloud Deploy Static Websites (Dev)
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)